lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1942,6 +1943,29 @@ LogicalResult StmtEmitter::visitSV(AliasOp op) {
   return success();
 }
 
+LogicalResult StmtEmitter::visitSV(InterfaceInstanceOp op) {
```

Comment:
At some point we probably need to refactor interfaces as in large part they look like modules in instantiation.

Reply:
Agree. That was a refactor that I wanted while writing this.

Reply:
I opened #1464 to track this. Thanks for the suggestion.